complete/src/main/java/com/example/restservice/service/QueueService.java

```diff
@@ -31,13 +30,11 @@
   public CreateQueueResponse createQueue(CreateQueueRequest createQueueRequest) {
     try {
       var queue =
```

Comment:
Oh does this mean earlier the changes were not immediately propogaged to the db?
And is this to fix the duplicate name error because of that reason?

Reply:
Right, the exception was not caught in this catch block, since we were not flushing.

See https://stackoverflow.com/questions/54876448/how-to-catch-hibernate-constraintviolationexception-or-spring-dataintegrityviol